modules/server/src/main/ts/bedrock/compiler/Webpack.ts

```diff
@@ -32,11 +32,29 @@ const moduleAvailable = (name: string) => {
 
 const webpackRemap: Array<Record<string, any>> = moduleAvailable('@ephox/swag') ? [
   {
-    test: /\.js|\.tsx?$/,
+    test: /\.js|\.mjs|\.tsx?$/,
```

Comment:
I realise this isn't related to your changes, but should this be:
```suggestion
    test: /(\.js|\.mjs|\.tsx?)$/,
```
or even
```suggestion
    test: /\.(js|mjs|tsx?)$/,
```

So that things like `file.mjs-other-extension.html` don't match? It's out of scope, so feel free to ignore, but it looks like a few other regexes in this file have the same problem and it should be an easy fix if you want to do it.

Reply:
ah, yeah that's probably what was intended.